### PR TITLE
DEV: save tiff and txt

### PR DESCRIPTION
**Pull request by licode**
in progress

### CODE REVIEW

File: pyxrf/model/fit_spectrum.py [hunk 1]
```diff
@@ -473,7 +475,7 @@ def fit_single_pixel(self):
         This function performs single pixel fitting.
         Multiprocess is considered.
         """
-        save_tiff = True
+        #save_tiff = True
```
Comment (ericdill) [inline pyxrf/model/fit_spectrum.py]: just delete this code. don't leave the commented code in

File: pyxrf/model/fit_spectrum.py [hunk 2]
```diff
@@ -558,10 +560,15 @@ def fit_single_pixel(self):
                             output_folder, use_sinp=use_snip)
             logger.info('Done with saving fitting plots.')
 
-        if save_tiff == True:
+        if self.save_tiff == True:
```
Comment (ericdill) [inline pyxrf/model/fit_spectrum.py]: you can do this with just `if self.save_tiff`, don't need the `== True`

File: pyxrf/model/fit_spectrum.py [hunk 3]
```diff
@@ -558,10 +560,15 @@ def fit_single_pixel(self):
...
             namelist = self.data_title.split('_')
             output_n = namelist[0]+'_'+namelist[1]+'_'+'output'
             output_data(fpath, os.path.join(self.result_folder, output_n))
+        if self.save_txt == True:
```
Comment (ericdill) [inline pyxrf/model/fit_spectrum.py]: Same here. drop the `== True`

File: pyxrf/view/fit.enaml [hunk 4]
```diff
@@ -683,7 +692,26 @@ enamldef FitView(DockItem): fit_view:
...
+            CheckBox: cb_txt:
+                text = 'Save to txt'
+                checkable = True
+                #minimum_size = (20, 20)
+                checked := fit_model.save_txt
+                clicked ::
+                    if checked:
+                        fit_model.save_txt = True
+                    else:
+                        fit_modle.save_txt = False
```
Comment (ericdill) [inline pyxrf/view/fit.enaml]: typo here. i assume you mean `fit_model`, not `fit_modle`

File: pyxrf/view/fit.enaml [hunk 5]
```diff
@@ -683,7 +692,26 @@ enamldef FitView(DockItem): fit_view:
...
                     img_model_rgb.data_dict_keys = []
                     img_model_rgb.data_dict_keys = img_model_rgb.data_dict.keys()
-
+            CheckBox: cb_tiff:
+                text = 'Save to tiff'
+                checkable = True
+                #minimum_size = (20, 20)
+                checked := fit_model.save_tiff
+                clicked ::
+                    if checked:
```
Comment (ericdill) [inline pyxrf/view/fit.enaml]: You can turn this block into `fit_model.safe_tiff = checked`

Comment (ericdill) [inline pyxrf/view/fit.enaml]: Actually, isn't this already being done in the line `checked := fit_model.save_tiff`?

Comment (licode) [inline pyxrf/view/fit.enaml]: good catch. I will remove those.

**Pull request merged**